### PR TITLE
3 add jest and testing library

### PR DESCRIPTION
**Pull request by Atzac**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build-storybook": "build-storybook",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
-    "prettier": "prettier --write ."
+    "prettier": "prettier --write .",
+    "test": "jest"
```
Comment (AlyssonMascarenhas) [inline package.json:12]:
```suggestion
    "test": "jest",
    "test:coverage": "jest -i --coverage",
    "test:watch": "jest -i --watchAll",
```

Comment (AlyssonMascarenhas) [reply package.json:12]: Quando adicionar o coverage, inserir no .gitignore e no .prettierignore

**Review by AlyssonMascarenhas — APPROVED** [review]
Congratulations on the job! excellent contribution ✅

**Review by iurynogueira — APPROVED** [review]

**Review by Deeved — APPROVED** [review]

**Review by AlyssonMascarenhas — APPROVED** [review]
Just resolve conflicts and run to embrace 😄

**Review by iurynogueira — APPROVED** [review]

**Pull request merged**